hooks/ios/iOSCopyPreferences.js

```diff
@@ -7,14 +7,12 @@ module.exports = function (context) {
     var projectRoot = context.opts.cordova.project ? context.opts.cordova.project.root : context.opts.projectRoot;
     var configXML = path.join(projectRoot, 'config.xml');
     var configParser = new ConfigParser(configXML);
-    var enableAppTracking = configParser.getPlatformPreference("EnableAppTrackingTransparencyPrompt", "ios");
-
-    var appNamePath = path.join(projectRoot, 'config.xml');
-    var appNameParser = new ConfigParser(appNamePath);
-    var appName = appNameParser.name();
+    
+    var appName = configParser.name();
```

Comment:
we can replace all `var` variables with `let`, like it's in `androidCopyPreferences.js`

Reply:
Of course! Can you check again?